components/eamxx/src/diagnostics/tests/extraaci_test.cpp

```diff
@@ -0,0 +1,184 @@
+#include "catch2/catch.hpp"
+#include "diagnostics/register_diagnostics.hpp"
+#include "pio.h"
+#include "share/field/field_utils.hpp"
+#include "share/grid/mesh_free_grids_manager.hpp"
+#include "share/io/scorpio_input.hpp"
+#include "share/util/scream_setup_random_test.hpp"
+
+namespace scream {
+
+std::shared_ptr<GridsManager> create_gm(const ekat::Comm &comm, const int ncols,
+                                        const int nlevs) {
+  const int num_global_cols = ncols * comm.size();
+
+  using vos_t = std::vector<std::string>;
+  ekat::ParameterList gm_params;
+  gm_params.set("grids_names", vos_t{"Point Grid"});
+  auto &pl = gm_params.sublist("Point Grid");
+  pl.set<std::string>("type", "point_grid");
+  pl.set("aliases", vos_t{"Physics"});
+  pl.set<int>("number_of_global_columns", num_global_cols);
+  pl.set<int>("number_of_vertical_levels", nlevs);
+
+  auto gm = create_mesh_free_grids_manager(comm, gm_params);
+  gm->build_grids();
+
+  return gm;
+}
+
+TEST_CASE("extraaci_dummy") {
+  using namespace ShortFieldTagsNames;
+  using namespace ekat::units;
+  // A world comm
+  ekat::Comm comm(MPI_COMM_WORLD);
+
+  scorpio::init_subsystem(comm);
+
+  std::string iop_file =
+      "/global/cfs/cdirs/e3sm/inputdata/atm/cam/scam/iop/"
+      "DYCOMSrf02_iopfile_4scam.nc";
```

Comment:
You can pass input args to your unit test. In your CMakeLists.txt, do
```cmake
GetInputFile(cam/scam/iop/DYCOMSrf02_iopfile_4scam.nc)
CreateDiagTest(extraaci "extraaci_test.cpp"
  EXE_ARGS "--ekat-test-params data=${SCREAM_INPUT_ROOT}//atm/cam/scam/iop/DYCOMSrf02_iopfile_4scam.nc)
```
and in the cpp file for the test, you can do
```cpp
const auto& data = ekat::TestSession::get().params.at("data");
```
You can use whatever string you want instead of `data`, so long as it's the same in the cpp file and in CMakeLists.txt.